wfsim/strax_interface.py

```diff
@@ -311,5 +309,5 @@ def compute(self, chunk_i):
             result = next(self.sim_iter)
         except StopIteration:
             raise RuntimeError("Bug in chunk count computation")
-        # self._sort_check(result['raw_records'])
+        self._sort_check(result['raw_records'])
```

Comment:
Do you know why there should be a gap in between chunks? Currently, the result from ChunkRawRecords are already guaranteed to be sorted in time.

Reply:
Strax expects a minimum time distance between chunks. For details why we need @JelleAalbers.
Currently the chunks are sorted, however the signals on the border gets written to two seperate chunks. This is not what we want

Reply:
Strax expects that it can find peaks independently in each chunk, and peaks may never overlap in strax. If there is no gap between chunks, the peaks can overlap (depending on the left and right extension set in the peakfinder / hitfinder).